doc/source/whatsnew/v0.25.0.rst

```diff
@@ -172,7 +172,7 @@ Plotting
 Groupby/Resample/Rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
--
+- Fixed bug in GroupBy when using as_index=False (issue:`25011`)
```

Comment:
Bug in ::meth:`pandas.core.groupby.GroupBy.agg`` when passing ``as_index=False``, returning an additional column.